### PR TITLE
log cfgzErr if err happened

### PR DESCRIPTION
**Pull request by NickrenREN**
We need to log err info when err info returned by initConfigz(),no matter what the result of utilconfig.DefaultFeatureGate.DynamicKubeletConfig() is and
whether s.RunOnce is true or not.
We should log the initKubeletConfigSync() err info too.

### CODE REVIEW

**Comment by NickrenREN** [comment]
@mtaufen  Could you spend some time helping to ask k8s-bot to test and review ? Thanks

**Comment by mtaufen** [comment]
Sure, I'll take a look at this right now. Just got to it after post-vacation catch-up.

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -353,8 +353,14 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 				if err != nil {
 					return err
 				}
+			} else {
+				glog.Warningf("failed to init kubelet configure sync : %v", err)
```
Comment (mtaufen) [inline cmd/kubelet/app/server.go]: nit: can you reword this to `failed to init dynamic Kubelet configuration sync: %v`, and make it an error instead of a warning?

File: cmd/kubelet/app/server.go [hunk 2]
```diff
@@ -353,8 +353,14 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
...
 			}
+		} else if cfgzErr != nil {
```
Comment (mtaufen) [inline cmd/kubelet/app/server.go]: This additional logging is not necessary. Errors during the configz setup are already logged by the `initConfigz` function. If the "unable to register configz" error is unclear, feel free to reword it.

**Review by mtaufen — CHANGES_REQUESTED** [review]

**Comment by NickrenREN** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[cmd/kubelet/app/server.go, line 357 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39370#-K_Zf53G595hy_FDQ8fa-r1-357:-KaABIeDa1AClfV_iVJX:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/95f3a5b640c1b1730d879eef7fb034e7eb0efc91/cmd/kubelet/app/server.go#L357)):*
<details><summary><i>Previously, mtaufen (Michael Taufen) wrote…</i></summary><blockquote>

nit: can you reword this to `failed to init dynamic Kubelet configuration sync: %v`, and make it an error instead of a warning?
</blockquote></details>

Done.

---

*[cmd/kubelet/app/server.go, line 359 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39370#-K_Zf53G595hy_FDQ8fa-r1-359:-KaABJ4pg9gxF8oWt8Yn:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/95f3a5b640c1b1730d879eef7fb034e7eb0efc91/cmd/kubelet/app/server.go#L359)):*
<details><summary><i>Previously, mtaufen (Michael Taufen) wrote…</i></summary><blockquote>

This additional logging is not necessary. Errors during the configz setup are already logged by the `initConfigz` function. If the "unable to register configz" error is unclear, feel free to reword it. 
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39370)*

**Comment by NickrenREN** [comment]
@mtaufen   Thanks for you review and advice. I have modified. PTAL

File: cmd/kubelet/app/server.go [hunk 3]
```diff
@@ -356,6 +356,8 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 				if err != nil {
 					return err
 				}
+			} else {
+				glog.Errorf("failed to init dynamic Kubelet configuration sync : %v", err)
```
Comment (mtaufen) [inline cmd/kubelet/app/server.go]: Thanks for the updates, snip the space between "sync" and ":" and I'll LGTM.

**Comment by NickrenREN** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[cmd/kubelet/app/server.go, line 360 at r2](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39370#-K_Zf53G595hy_FDQ8fa-r2-360:-KaRETdjFJEnACOpe7M5:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/66da1545ea99508646989a3128d105bab1ae2e02/cmd/kubelet/app/server.go#L360)):*
<details><summary><i>Previously, mtaufen (Michael Taufen) wrote…</i></summary><blockquote>

Thanks for the updates, snip the space between "sync" and ":" and I'll LGTM.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39370)*

**Comment by NickrenREN** [comment]
@mtaufen  Thanks for you review. I have snipped the space. PTAL

**Review by mtaufen — APPROVED** [review]

**Comment by mtaufen** [comment]
Thanks for this work :)

**Comment by NickrenREN** [comment]
@mtaufen  Sorry to bother you. Do you have time to help add  "release-note" label .  Thanks

**Comment by mtaufen** [comment]
Sure thing.
